From 777cf32f09f7214fb7b422288d9a5921633106b5 Mon Sep 17 00:00:00 2001
From: Daniel Ammann <daniel.ammann@bytesatwork.ch>
Date: Fri, 17 Jul 2020 11:05:52 +0200
Subject: [PATCH] stmp32mp1: Append target bytedevkit

---
 arch/arm/mach-stm32mp/Kconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/mach-stm32mp/Kconfig b/arch/arm/mach-stm32mp/Kconfig
index f9f79437e4..80274830c3 100644
--- a/arch/arm/mach-stm32mp/Kconfig
+++ b/arch/arm/mach-stm32mp/Kconfig
@@ -77,6 +77,13 @@ config TARGET_ST_STM32MP15x
 		Evalulation board (EV1) or Discovery board (DK1 and DK2).
 		The difference between board are managed with devicetree
 
+config TARGET_BYTEDEVKIT
+	bool "bytesatwork byteDEVKIT"
+	select STM32MP15x
+	imply BOOTCOUNT_LIMIT
+	help
+		Enable byteDEVKIT as target.
+
 config TARGET_DH_STM32MP1_PDK2
 	bool "DH STM32MP1 PDK2"
 	select STM32MP15x
-- 
2.20.1

