From d134e02d02d9db77b3fe75f569c28b3a7575586f Mon Sep 17 00:00:00 2001
From: Lothar Rubusch <lothar.rubusch@bytesatwork.ch>
Date: Mon, 22 Nov 2021 11:19:21 +0100
Subject: [PATCH] bytedevkit: setup single rootfs partition

Signed-off-by: Lothar Rubusch <lothar.rubusch@bytesatwork.ch>
---
 include/configs/bytedevkit.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/configs/bytedevkit.h b/include/configs/bytedevkit.h
index 230c9eb780..753ccba02b 100644
--- a/include/configs/bytedevkit.h
+++ b/include/configs/bytedevkit.h
@@ -96,9 +96,9 @@
  */
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	"mmc_dev=0:4\0" \
-	"mmc_root=/dev/mmcblk0p5\0" \
-	"dtbfile=stm32mp157c-bytedevkit.dtb\0" \
-	"kernelfile=uImage\0" \
+	"mmc_root=/dev/mmcblk0p4\0" \
+	"dtbfile=/boot/stm32mp157c-bytedevkit.dtb\0" \
+	"kernelfile=/boot/uImage\0" \
 	"loadaddr=0xc1000000\0" \
 	"stdin=serial\0" \
 	"stdout=serial\0" \
-- 
2.20.1

